tools/harness-automation/autothreadharness/settings_sample.py

```diff
@@ -73,9 +77,14 @@
 
 GOLDEN_DEVICES = []
 """[(str, str)]: devices list.
-
 It should be something like [('COM1', 'OpenThread'), ('COM2', 'ARM')] for devices connected to Windows.
+For OpenThread golden devices, ser2net is also supported, just use IP:PORT for the name. For example,
+('192.168.1.2:5001', 'OpenThread').
+"""
 
+SHIELD_GOLDEN_DEVICES = []
```

Comment:
For IV, how to know how many and what kind of devices should be put in the shield box? any guidance?

For easier adoption, better to either provide the suggested setting for specific topologyconfig or provide a tool to analyze the topologyconfig and give suggestion?